### DISCUSSION TITLE
As an iOS user on iPad, I need views with columns to be supported, so I can manage the stages the project tickets are in without scrolling.

### DISCUSSION THREAD

**Discussion by PeteSmalls** [community/community#61040] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.121.0 (104034577)
Device: iPad Pro 11-inch 2 (iOS 16.3.1)
</details>

**Comment by esahmed21** [comment]
Hey @PeteSmalls 👋 

Thank you so much for reaching out to us! I'll open a feature request issue to track it and discuss internally with the team! Thanks for the feedback.!
